### DISCUSSION TITLE
Improve debugging experience for minified / obfuscated code

### DISCUSSION THREAD

**Discussion by bhackett1024** [RecordReplay/devtools#5529] [General]
This is a low priority wishlist type feature, but it can be hard at times to understand what code is executing when it has been minified and/or obfuscated.

For example, the recording below is one I made of yarn, to see why `yarn jest` worked in a package that did not include jest as a script in its package.json.  The version of yarn I'm using has minified source, and while I was able to find the place where it spawned the subprocess and passed in the "jest" script to run, figuring out where that script path came from eventually led to a wad of `&&` and `,` and `await` expressions chained together, and I gave up.  It would be really neat to pick apart the control flow here and understand what this code is doing.

https://app.replay.io/recording/replay-of-yarn-311cjs--53cbe7d6-f6d2-4a96-9d73-64373082b18f?point=1947111326353493799816159821825554&time=2133.3545232273837&hasFrames=true#

**Comment by jasonLaster** [comment]
Moving this to a discussion and including some more folks

Curious @narendrasss + @geoffreylitt if you have thoughts on this

For me, providing a simple way to edit minified variables would be useful. For instance, changing `u` to `jest` would trigger a babel transform that updated the correct references to `u`.
